### ISSUE TITLE
Can't scroll on mobile

### ISSUE THREAD

**Issue by chrisjbrown** [bryaneaton13/react-pull-to-refresh#12]
On a mobile device or in chrome dev tools emulating a device I cannot scroll. Scrolling up does nothing and scrolling down activates the pull to refresh.

**Comment by simonxeko** [comment]
I stumbled across this library yesterday and tried to use it on our web app. 
This seems to be a **hammerjs** ( the touch control library used by this ) problem. If you use inspector to cancel the css props on .refresh-view this component generated ( user-select, user-drag, touch-action ), you will be able to scroll again, but it might become a little buggy in desktop browsers. 

So to use this lib you might need to fork your own hammerjs or request hammerjs to provide an option to disable those css props.

**Comment by goshcc** [comment]
How to reslove

**Comment by XSlemX** [comment]
@goshcc, as @simonxeko points out, this is a due to the default touch-action in hammerjs is 'none'. A fix could be to set the default to 'pan-y'. Like this: 
`let hammer = require('hammerjs');
hammer.defaults.touchAction = 'pan-y';`

This might not be the best solution, but will fix your immediate problem.

EDIT: Ignore proposed solution. Will break how the component works.

**Comment by aplanchamp** [comment]
@bryaneaton13 hello :) any idea if and when this issue will be fixed ?
Thanks

**Comment by robinsonlam** [comment]
hi, has anyone addressed this issue?

**Comment by KevenGarnet** [comment]
I also encountered the same problem, I ask you to solve it?

**Comment by idesignpixels** [comment]
This solution breaks the pull to fresh behaviour on mine.

**Comment by CruxCv** [comment]
nanshou

**Comment by JTorr** [comment]
Has anyone come up with a solution to this?

**Comment by JordanForeman** [comment]
bump

**Comment by princewck** [comment]
I forked this repo and made a hack to solve this, if this hack solution may help u .
https://www.npmjs.com/package/react-pull2refresh

**Comment by hectorhottomali** [comment]
I tried your repo @princewck, but I still have an issue, when I pull down the handler does get called, however the issue is with the scroll, so if scroll down on a screen when you try to scroll back up or pulling the screen down just to move the page the handler gets called and I cant move the screen "up".

To scroll correctly I need to "pull up" a little bit and then "pull down/scroll" (all in one touch movement).

**Comment by princewck** [comment]
@hectorhottomali  yes, I finally tried other alternatives and [pulltorefreshjs](https://www.npmjs.com/package/pulltorefreshjs) this package solved my problem.

**Comment by michaelpeterlee** [comment]
We experience the same problem on iOS and Android web view.

**Comment by michaelpeterlee** [comment]
https://www.npmjs.com/package/pulltorefreshjs also does not work with Cordova web-view. 
Folks may like to look at how Ionic V1 solved this problem.

**Comment by s10mcow** [comment]
Isnt pull to refresh typically thought of as a mobile thing, and it doesnt work on mobile... sofaking confused.

**Comment by redclouddrailan** [comment]
use this https://thmsgbrt.github.io/react-simple-pull-to-refresh/ this works on mobile too.
